### ISSUE TITLE
MO version 2.0.6 crash running Fallout or FO4Edit after modifying esp with FO4edit

### ISSUE THREAD

**Issue by zeruel11** [TanninOne/modorganizer#791]
I don't know if this is a bug with me or is intended for now, but my installation of MO always crash trying to run basically any programs (Fallout, F4SE, FO4Edit) after I uses FO4Edit to modify an esp file. If that was vague, what I did is, open MO -> run FO4Edit from MO -> edit esp in FO4Edit -> exit FO4Edit and save esp -> run F4SE (or FO4Edit again since I forgot something). The last one always results in MO crash.
I also uses the create files in mod instead of overwrite feature to route the esp to a dummy mod since  as far as I know MO's overwrite function doesn't work yet. Is that the problem?
Here is the zipped dump file of the crash, if it helps. Thank you.
[MO Dump.zip](https://github.com/TanninOne/modorganizer/files/305082/MO.Dump.zip)

**Comment by michaelrw10** [comment]
Yes.. I also receive A LOT of crashes ... it is a bit annoying, but other than that, it is benign. Just restart MO and keep doing what you were doing. It is a small inconvenience and a small price to for all the great things that MO does. 

For the record, I use Win 10, as well. But, no way of knowing if that is related to the crashes.

**Comment by zeruel11** [comment]
Well, I actually posted this more as a heads up or testing report, since I didn't find other issue about MO crashes this way. Or maybe I'm just being annoying and expecting that MO2 will be very stable at this point of development, haha. Thanks for the answer though, will patiently wait for MO2 to be more stable.
As for the win 10 part, I was referring to F4SE 'failed to identify exe type' problem, not the reason for crashes. Somehow win 10 doesn't handle Run as admin the same way as running said program from an elevated cmd.
